Makefile

```diff
@@ -433,7 +434,14 @@ TARGETS = \
 	installers-all \
 	electron-develop
 
-package-electron: $(BUILD_DIRECTORY)/$(APPLICATION_NAME)-$(APPLICATION_VERSION)-$(TARGET_PLATFORM)-$(TARGET_ARCH)
```

Comment:
The Linux rules still use that target. We will be able to remove it after merging the Linux electron-builder PR.